system/Security/CheckPhpIni.php

```diff
@@ -142,8 +142,8 @@ public static function checkIni(): array
 
         foreach ($items as $key => $values) {
             $output[$key] = [
-                'global'      => $ini[$key]['global_value'],
-                'current'     => $ini[$key]['local_value'],
+                'global'      => array_key_exists($key, $ini) ? $ini[$key]['global_value'] : 'disabled',
```

Comment:
array_key_exists same condition can be moved to before `$output[$key]` definition, eg:

```php
$hasKeyInIni = array_key_exists($key, $ini);
$output[$key] = [
```

so no need double call.

Reply:
Exactly. Done.